scripts/dovecot/README.md

```diff
@@ -0,0 +1,83 @@
+## Introduction to custom Dovecot builds
+
+Chatmail servers use a custom Debian build of the IMAP 'dovecot' server software because 
+
+a) Dovecot developers did not yet merge a [pull request](https://github.com/dovecot/core/pull/216)
+   which majorly speeds up message delivery by removing a hardcoded 0.5 second delay 
+   on relaying incoming messages.  
+
+b) Even if merged, it would take years for it to reach Debian stable,
+   the distribution that chatmail deployment uses. 
+
+c) The modified dovecot has been successfully used since December 2023 without issues
+   and we see no noticeable downside (theoretically higher CPU usage but not measureable)
+   but a considerable upside as the delay-removal facilitates end-to-end message 
+   delivery of 200 ms in real networks. 
+
+The modified forked dovecot code lives at 
+[https://github.com/chatmail/dovecot](https://github.com/chatmail/dovecot).  
+The remainder of this document describes the setup of the Debian repository 
+containing the patched dovecot version. 
+
+## Building Debian packages at build.opensuse.org 
+
+Deltachat developers maintain a [shared account](https://build.opensuse.org/project/show/home:deltachat) 
```

Comment:
```suggestion
Delta Chat developers maintain an [account](https://build.opensuse.org/project/show/home:deltachat) 
```

Otherwise I'd wonder "shared with who"? And it is actually just shared between the sysadmin team.

Reply:
well, I added this, because it is not the recommended way of working with the obs - it is a bit like sharing a github-account for our team

Reply:
but we don't have to make it public that we don't adhere to suse best practices